dataflow/manual/content.tex

```diff
@@ -299,6 +299,31 @@ \subsubsection{AbstractValue}
 class NullnessValue extends CFAbstractValue<NullnessValue>
 \end{verbatim}
 
+\subsubsection{UnusedAbstractValue}
+\label{sec:UnusedAbstractValue_classes}
+
+UnusedAbstractValue, as the name suggests, is an AbstractValue but is
```

Comment:
Maybe `NoMergeAbstractValue` since it sounds more descriptive to me.

Reply:
Sure, I think that's a descriptive name, I will talk with Prof about it on Monday

Reply:
This class should only be used as a type argument in transfer functions. In this first paragraph, the important point isn't what LUB computes, it is why we have this class and when to use it.

Reply:
added.